{"uuid": "a6844a27-a53a-42a3-8280-50daaaff3213", "children": ["b10d177d-86d7-4de5-bb48-bea11a73b381"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: unknown error: cannot find Chrome binary\nStacktrace:\n0   chromedriver                        0x0000000100a27f48 chromedriver + 4226888\n1   chromedriver                        0x0000000100a204f4 chromedriver + 4195572\n2   chromedriver                        0x0000000100664d68 chromedriver + 281960\n3   chromedriver                        0x000000010068bc24 chromedriver + 441380\n4   chromedriver                        0x000000010068a150 chromedriver + 434512\n5   chromedriver                        0x00000001006c793c chromedriver + 686396\n6   chromedriver                        0x00000001006c7164 chromedriver + 684388\n7   chromedriver                        0x0000000100693f1c chromedriver + 474908\n8   chromedriver                        0x0000000100694ef4 chromedriver + 478964\n9   chromedriver                        0x00000001009e959c chromedriver + 3970460\n10  chromedriver                        0x00000001009ed6f0 chromedriver + 3987184\n11  chromedriver                        0x00000001009f35b4 chromedriver + 4011444\n12  chromedriver                        0x00000001009ee2fc chromedriver + 3990268\n13  chromedriver                        0x00000001009c61c0 chromedriver + 3826112\n14  chromedriver                        0x0000000100a0a088 chromedriver + 4104328\n15  chromedriver                        0x0000000100a0a1e0 chromedriver + 4104672\n16  chromedriver                        0x0000000100a19f28 chromedriver + 4169512\n17  libsystem_pthread.dylib             0x0000000184c6ffa8 _pthread_start + 148\n18  libsystem_pthread.dylib             0x0000000184c6ada0 thread_start + 8\n\n", "trace": "  File \"/Users/sokolov/Documents/qa-automation-learning/qa-automation-learning/.venv/lib/python3.13/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/Users/sokolov/Documents/qa-automation-learning/qa-automation-learning/.venv/lib/python3.13/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"/Users/sokolov/Documents/qa-automation-learning/qa-automation-learning/.venv/lib/python3.13/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/Users/sokolov/Documents/qa-automation-learning/qa-automation-learning/.venv/lib/python3.13/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/sokolov/Documents/qa-automation-learning/qa-automation-learning/.venv/lib/python3.13/site-packages/_pytest/fixtures.py\", line 1195, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/sokolov/Documents/qa-automation-learning/qa-automation-learning/.venv/lib/python3.13/site-packages/_pytest/fixtures.py\", line 922, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/Users/sokolov/Documents/qa-automation-learning/qa-automation-learning/conftest.py\", line 9, in driver\n    driver = webdriver.Chrome(options=options, service=ChromeService(ChromeDriverManager().install()))\n  File \"/Users/sokolov/Documents/qa-automation-learning/qa-automation-learning/.venv/lib/python3.13/site-packages/selenium/webdriver/chrome/webdriver.py\", line 47, in __init__\n    super().__init__(\n    ~~~~~~~~~~~~~~~~^\n        browser_name=DesiredCapabilities.CHROME[\"browserName\"],\n        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n    ...<3 lines>...\n        keep_alive=keep_alive,\n        ^^^^^^^^^^^^^^^^^^^^^^\n    )\n    ^\n  File \"/Users/sokolov/Documents/qa-automation-learning/qa-automation-learning/.venv/lib/python3.13/site-packages/selenium/webdriver/chromium/webdriver.py\", line 69, in __init__\n    super().__init__(command_executor=executor, options=options)\n    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/sokolov/Documents/qa-automation-learning/qa-automation-learning/.venv/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 261, in __init__\n    self.start_session(capabilities)\n    ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^\n  File \"/Users/sokolov/Documents/qa-automation-learning/qa-automation-learning/.venv/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 362, in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n               ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/sokolov/Documents/qa-automation-learning/qa-automation-learning/.venv/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 454, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/Users/sokolov/Documents/qa-automation-learning/qa-automation-learning/.venv/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1754146066633, "stop": 1754146076714}], "start": 1754146066633, "stop": 1754146076762}